docs/API_SPECIFICATION.md

```
@@ -64,6 +64,12 @@
 | **12. 공통 - 설정** ||||||||
 | 12.1 | 내 설정 조회 | GET | `/api/settings/me` | 로그인한 사용자의 알림 설정 조회 | 대기 | 완료 | 2025-12-10 | [링크](#121-내-설정-조회) |
 | 12.2 | 내 설정 수정 | PUT | `/api/settings/me` | 로그인한 사용자의 알림 설정 수정 | 대기 | 완료 | 2025-12-10 | [링크](#122-내-설정-수정) |
+| **13. 공통 - 공지사항** ||||||||
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 9; Actual: 8; Too few cells, row will be missing data

(MD056, table-column-count)